lib/URI/icaps.pm

```diff
@@ -0,0 +1,11 @@
+package URI::icaps;
+
+use strict;
+use warnings;
+use base qw(URI::icap);
+
+our $VERSION = 0.07;
```

Comment:
This should probably be `our $VERSION = '5.20';`

Not sure if it actually matters, though. 🤔